docs/sources/breaking-changes/breaking-changes-v11-0.md

```diff
@@ -0,0 +1,220 @@
+
+For more details on the code removal, review the following PRs:
+
+- [https://github.com/grafana/grafana/pull/83651](https://github.com/grafana/grafana/pull/83651)
+- [https://github.com/grafana/grafana/issues/81268](https://github.com/grafana/grafana/issues/81268)
+
+### Deprecated endpoints and fields in Reporting removed
+
+#### Description
+
+In Grafana v11, support for deprecated endpoints and fields in **Reporting** related to the old scheduling format, email, and dashboard is fully removed. This prevents any calls to deprecated endpoints and passing in values to deprecated fields. This feature only affects Cloud and Enterprise customers who use the API to generate reports.
+
+#### Migration/mitigation
+
+Ensure deprecated endpoints are updated to new corresponding endpoints and deprecated fields are removed and replaced with new corresponding fields.
+
+#### Learn more
+
+The [Reporting documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/developers/http_api/reporting/) lists all supported endpoints and fields.
+
+### Change custom branding public dashboard footer behavior
+
+#### Description
+
+In Grafana v11, custom branding public dashboard footer behavior is changed to default to the Grafana logo if no footer logo or footer text is set. There is no option to hide the public dashboard footer anymore. This feature only affects Cloud Advanced and Enterprise customers.
+
+#### Migration/mitigation
+
+Ensure you have a public dashboard footer logo or footer text set if you don't want to display the default Grafana footer.
+
+#### Learn more
+
+[Configure custom branding documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/configure-custom-branding/#custom-branding-for-public-dashboards) for public dashboards
+
+### Subfolders cause very rare issues with folders that have forward slashes in their names
+
+#### Description and migration/mitigation
+
+The upgrade to enable subfolders can cause some issues with alerts in certain cases. If you've previously set up a folder that uses a forward slash in its name, you have an alert rule in that folder, and the notification policy is set to match that folder's name, notifications will be sent to the default receiver instead of the configured receiver.
+
+In these cases, we recommend that you take these steps before the upgrade to enable subfolders:
+
+- Create a copy of the affected routes and rewrite the matchers for the new copy. For example, if the original matcher was `grafana_folder=MyFolder/sub-folder`, then the new route matcher will be `grafana_folder=MyFolder\/sub-folder`.
+- After enabling subfolders, you can delete the old routes.
+
+Please note that if you use file provisioning, you can upgrade and update the routes at the same time.
+
+#### Learn more
+
+[Subfolders announcement](https://grafana.com/docs/grafana-cloud/whats-new/#subfolders)
```

Comment:
This links to the Grafana Cloud post which is probably the same as this isn't it?

Reply:
Good catch, thank you!

Reply:
Oh I just realized that this is the breaking changes doc which is different. I thought it was the What's new! 🤦

Reply:
Yeah I think that blog post does have more content. Maybe the link should be to the Grafana docs though?

Reply:
LOL! Whoops!